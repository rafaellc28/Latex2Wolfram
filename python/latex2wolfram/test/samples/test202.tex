% Test generic math expression
(\sum_{j = 0}^{\infty}\left(\frac{3}{4}\right)^{j}) + \int_{0}^{2}x^{2} * (\sin x)^{3}dx
