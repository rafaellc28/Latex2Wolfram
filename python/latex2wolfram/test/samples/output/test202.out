((sum((3/4) ^ j), from j=0 to infinity)) + (integrate x ^ 2 * (sin(x)) ^ 3 dx from x=0 to 2)
